<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1980</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="StartingScene">
       <widget class="QPushButton" name="StartingSceneNewGame">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>342</y>
          <width>280</width>
          <height>100</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{ 
        background-color: transparent;
        border: none;
        background-repeat: none;
        background: none;
        background-image: url(:/greenLay280x100.png);
        font-size: 22px;
        font-family: Comic Sans MS;
        color: white;
    }
   	QPushButton:hover{
        background-image:url(:/greenLay280x100Highlighted.png);
    };</string>
        </property>
        <property name="text">
         <string>New game</string>
        </property>
       </widget>
       <widget class="QPushButton" name="StartingSceneLoadGame">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>462</y>
          <width>280</width>
          <height>100</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{ 
        background-color: transparent;
        border: none;
        background-repeat: none;
        background: none;
        background-image:url(:/greenLay280x100.png);
        font-size: 22px;
        font-family: Comic Sans MS;
        color: white;
    }
    :hover{
        background-image:url(:/greenLay280x100Highlighted.png);
    };</string>
        </property>
        <property name="text">
         <string>Load game</string>
        </property>
       </widget>
       <widget class="QPushButton" name="StartingSceneSettingsButton">
        <property name="geometry">
         <rect>
          <x>820</x>
          <y>582</y>
          <width>280</width>
          <height>100</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{ 
        background-color: transparent;
        border: none;
        background-repeat: none;
        background: none;
        background-image:url(:/greenLay280x100.png);
        font-size: 22px;
        font-family: Comic Sans MS;
        color: white;
    }
    :hover{
        background-image:url(:/greenLay280x100Highlighted.png);
    };

</string>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="NewGameScene">
       <widget class="QComboBox" name="NewGameSceneLeaguesComboBox">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>30</y>
          <width>1000</width>
          <height>100</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox{
				background-color: transparent;
                border: none;
                background-repeat: none;
                background: none;
                font-size: 35px;
                font-family: Comic Sans MS;
                color: black;
				text-align: center;
}

QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}

QComboBox QAbstractItemView {
    border: 2px solid darkgray;
    selection-background-color: lightgray;
}</string>
        </property>
       </widget>
       <widget class="QLabel" name="NewGameSceneClubName">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>160</y>
          <width>1000</width>
          <height>100</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
                background-color: transparent;
                border: none;
                background-repeat: none;
                background: none;
                font-size: 35px;
                font-family: Comic Sans MS;
                color: white;
				text-align: center;
            };
</string>
        </property>
        <property name="text">
         <string>CLUB NAME </string>
        </property>
       </widget>
       <widget class="QLabel" name="NewGameSceneClubLogo">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>290</y>
          <width>1000</width>
          <height>200</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="NewGameSceneClubInfoLabel">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>500</y>
          <width>1000</width>
          <height>180</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel{
                background-color: transparent;
                border: none;
                background-repeat: none;
                background: none;
                font-size: 25px;
                font-family: Comic Sans MS;
                color: white;
            };</string>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QPushButton" name="NewGameSceneStartButton">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>700</y>
          <width>400</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{ 
            background-color: transparent;
            border: none;
            background-repeat: none;
            background: none;
            background-image:url(:/greenLay400x120.png);
            font-size: 40px;
            font-family: Comic Sans MS;
            color: white;
        }
        :hover{
            background-image:url(:/greenLay400x120Highlighted.png);
        };
</string>
        </property>
        <property name="text">
         <string>Start game!</string>
        </property>
       </widget>
       <widget class="QPushButton" name="NewGameScenePrevClubButton">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>160</y>
          <width>120</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
                background-color: transparent;
                border: none;
                background-repeat: none;
                background: none;
                background-image:url(:/leftArrow120x120.png);
            }
            :hover{
                background-image:url(:/leftArrowHighlighted120x120.png);
            };
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="NewGameSceneNextClubButton">
        <property name="geometry">
         <rect>
          <x>1560</x>
          <y>160</y>
          <width>120</width>
          <height>120</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
                background-color: transparent;
                border: none;
                background-repeat: none;
                background: none;
                background-image:url(:/rightArrow120x120.png);
            }
            :hover{
                background-image:url(:/rightArrowHighlighted120x120.png);
            };
</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
